apps/studio/schemas/person.ts

```diff
@@ -19,9 +20,74 @@ export default defineType({
       type: "string",
     }),
     defineField({
-      name: "linkedinURL",
-      title: "LinkedIn URL",
-      type: "url",
+      name: "email",
+      title: "Email",
+      type: "email",
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: "socials",
+      title: "Socials",
+      type: "array",
+      of: [
+        {
+          type: "object",
+          fields: [
+            defineField({
+              name: "platform",
+              title: "Platform",
+              type: "string",
+              options: {
+                list: ["LinkedIn", "Personal Website"],
```

Comment:
Consider changing to` [{title: "LinkedIn", value: "linkedin"}, {title: "Personal Website", value: "site"}]` so we don't have to do `if (platform == "Personal Website")` and can instead do `if (platform == "site")`.
This might be personal preference though, so I'll leave it up to you.

Reply:
This will also fix the capitalization issue:
<img width="337" alt="Arc 2023-06-19 at 16 54 09@2x" src="https://github.com/HackAtUCI/hack.ics.uci.edu/assets/41758627/f6736189-cf7f-465d-a619-3cd23ccde969">

Reply:
This is definitely better than having to check against a multi-word string. Added in [09d1d86](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/09d1d8696fd479b7a0ae77d1e6d1cd0219bddd72)

Reply:
Fixed in [`09d1d86`](https://github.com/HackAtUCI/hack.ics.uci.edu/pull/12/commits/09d1d8696fd479b7a0ae77d1e6d1cd0219bddd72)